NativeApp/src/components/Common/Caption/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { ErrorText } from './styles';
+
+const Caption = (props) => {
+  const { children } = props;
+
+  return (
+    <ErrorText {...props}>{children}</ErrorText>
+  );
+};
+
+Caption.propTypes = {
+  children: PT.string.isRequired,
```

Comment:
I think it's safer to use `PT.node` for any children, just on the off chance you need to pass a View containing multiple texts or something

Reply:
cool, i have made the suggested changes :)